### PR TITLE
Codacy - Commented Private Functions

### PR DESCRIPTION
**Pull request by 87johnboyle**
A couple of the codacy issues were around commenting of private functions. Two of which had pretty self explanatory names, so I see no harm in removing the preceding comments.

### CODE REVIEW

**Comment by faruktoptas** [comment]
Thank you @87johnboyle I think the filename should be kept as `ISSUE_TEMPLATE`
https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser

File: fancyshowcaseview/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.kt [hunk 1]
```diff
@@ -185,9 +185,6 @@ class FancyShowCaseView @JvmOverloads constructor(context: Context, attrs: Attri
         })
     }
 
-    /**
-     * Starts enter animation of FancyShowCaseView
-     */ @TargetApi(Build.VERSION_CODES.LOLLIPOP)
```
Comment (faruktoptas) [inline fancyshowcaseview/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.kt:190]: `@TargetApi` annotation should be there

**Review by faruktoptas — CHANGES_REQUESTED** [review]

File: fancyshowcaseview/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.kt (hunk 1 above)
Comment (87johnboyle) [reply fancyshowcaseview/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.kt:190]: Apologies, didn't spot that. Added back in and just made a fresh commit.

**Review by faruktoptas — APPROVED** [review]

**Comment by faruktoptas** [comment]
Thanks :)

**Pull request merged**